spec.html

```diff
@@ -21501,6 +21439,9 @@ <h1>Runtime Semantics: ClassElementEvaluation</h1>
     <emu-clause id="sec-runtime-semantics-classdefinitionevaluation" type="sdo" aoid="ClassDefinitionEvaluation">
       <h1>Runtime Semantics: ClassDefinitionEvaluation</h1>
       <p>With parameters _classBinding_ and _className_.</p>
+      <emu-note>
+        <p>For ease of specification, private methods and accessors are included alongside private fields in the [[PrivateElements]] slot of class instances. However, any given object has either all or none of the private methods and accessors defined by a given class. Implementations may choose to implement these using a strategy which does not require tracking each method or accessor individually. This differs from private fields: because field initializers can throw during class instantiation, an individual object may have some proper subset of the private fields of a given class.</p>
```

Comment:
add something like: For example, it suffices to make a brand for each class and install [at the appropriate point]. (+ more explanation of how that would work)

[We should call out that it was intentionally designed to allow this implementation strategy.]